CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+# dbt_pendo_source v0.5.0
+## 🚨 Breaking Changes 🚨:
```

Comment:
should we also note that the `event_key` surrogate keys now include _fivetran_id in the hashing formula as well? i feel like that's a breaking change

Reply:
Yes, that is a good point! I added to this release's changelog and also v0.4.0 since I missed the keys that were affected then as well.